react-common/components/share/ShareInfo.tsx

```diff
@@ -32,6 +32,19 @@ export interface ShareInfoProps {
     onClose: () => void;
 }
 
+function useWindowWidth() {
```

Comment:
can this not be done with CSS?

Reply:
No, this is needed in JavaScript. We are already conditionally rendering the embed button with css and that's causing problems with tab navigation. On live, if you shrink the screen to make the embed button go away and then try to tab past the `copy` button, you can't. You can get to the lower buttons with shift tabbing, but that shouldn't be something that's necessary. That's because the embed button is not displayed and the tab index isn't set correctly, causing problems. I tried just checking the width at the rendering of the button, but that means that if the width of the screen gets larger, the embed button's `enabled` state won't change, thus, this listener needs to be here.

Reply:
ah, I see. then the right fix here should actually be in the focus trap in react-common, it should skip elements that have display: none.

Reply:
[this function](https://github.com/microsoft/pxt/blob/master/react-common/components/controls/FocusTrap.tsx#L29) needs to filter out elements with display: none. that should fix this bug and not require any changes to the css

Reply:
I was having trouble with finding the right query selector, I will make this change when I get back.